SnomDeskphone/module.php

```diff
@@ -37,13 +37,16 @@ public function instanceIpExists(): bool
 
     public function MessageSink(int $TimeStamp, int $SenderID, int $Message, array $Data): void
     {
-        $fkeysSettings = json_decode($this->ReadPropertyString("FkeysSettings"), true);
-        $fkeysToUpdate = $this->getFkeysToUpdate($fkeysSettings, $SenderID, $Data);
-        $urls = $this->getUrls($fkeysToUpdate, $SenderID, $Data);
-
-        foreach ($urls as $url) {
-            $this->httpGetRequest($url);
-        }
+		//Daten nur ans Telefon senden, wenn sich die Variable verändert hat.
+		if ($Data[1]) {	
```

Comment:
In order to avoid code comments that somewhen could be outdated, assign the value to an human readable variable.

```suggestion
        $valueChanged = $Data[1];

        if ($valueChanged) {
```